packages/elements/src/components.d.ts

```diff
@@ -317,6 +317,10 @@ export namespace Components {
           * If true, an *optional* message is displayed if not required, otherwise a * marker is displayed if required
          */
         "showLabelHint"?: boolean;
+        /**
+          * Position the calendar inside the wrapper and inside the ino-datepicker.
+         */
+        "static"?: boolean;
```

Comment:
Probably it isn't necessary to provide an api to control the desired behavior. Thinking about the datepicker is suppose there will be no use case where a non static usage of the datepicker is useful. What do you think?

Reply:
I do not know if there is no use case for this. I will asking around if there is an use case for this.

Reply:
A possible issue is rendering the overlay above other content in deep nested scenarios. IMO it is necessary to control this behavior in favor of usage at different DOM-context